shared/k8s/debug.go

```diff
@@ -0,0 +1,22 @@
+package k8s
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func FetchValidatorPrivateKeys(clientset *kubernetes.Clientset) (map[string]string, error) {
+	var validatorKeysMap = make(map[string]string)
```

Comment:
Haven't seen `var ` in a while.

Option 1: `validatorKeysMap := `
Option 2: named params